File: problems/006/006.c
#include <stdio.h>

/* Copyright 2021 Melwyn Francis Carlo */

int main()
{
    int n = 100;
    int sum = (n * (n + 1)) / 2;
    long int square_of_sum = sum * sum;
    long int sum_of_square = (n * (n + 1) * ((2 * n) + 1)) / 6;

    printf("%ld\n", square_of_sum - sum_of_square);

    return 0;
}

File: problems/006/006.f90
PROGRAM F006

    ! Copyright 2021 Melwyn Francis Carlo

    IMPLICIT NONE

    INTEGER :: N, N_SUM, SQUARE_OF_SUM, SUM_OF_SQUARE

    N = 100
    N_SUM = (N * (N + 1)) / 2
    SQUARE_OF_SUM = N_SUM * N_SUM;
    SUM_OF_SQUARE = (N * (N + 1) * ((2 * N) + 1)) / 6;

    PRINT ('(I0)'), SQUARE_OF_SUM - SUM_OF_SQUARE

END PROGRAM F006
